### ISSUE TITLE
Add Reset Functionality

### ISSUE THREAD

**Issue by KamalAman** [xxhomey19/use-undo#6]
The use-undo hook should export a reset function to reset a initial state state and clear all future/past states

```javascript

case RESET:
  const { newPresent } = action;

  return {
    past: [],
    present: newPresent,
    future: [],
  };
```

**Comment by xxhomey19** [comment]
Feel free to send a PR for it, or I'll implement it later.

**Comment by xxhomey19** [comment]
It's available in version v0.2.0 now :)
